internal/pkg/manifest/validate.go

```diff
@@ -556,6 +556,9 @@ func (r RoutingRuleConfigOrBool) Validate() error {
 			missingField: "path",
 		}
 	}
+	if r.Enabled != nil {
```

Comment:
I find this method a little confusing to grasp because we set `Enabled = nil` when the `RoutingRuleConfiguration` is not empty.
I'd have expected `Enabled = nil` and `Enabled = aws.Bool(false)` to have the same effect.

What do you think of instead setting `Enabled = true` when we can decode the configuration? and this method becomes:
```go
if !aws.BoolValue(r.Enabled) {
    return nil
}
return r.RoutingRuleConfiguration.Validate()
```

Instead the `path` check can be moved inside of `func (r RoutingRuleConfiguration) Validate() error`